app/views/definitions/_definitions_form.html.erb

```diff
@@ -1,35 +1,30 @@
 <h3>Add a New Definition</h3>
-<%= form_with(model:@definition, url: [@entry, @definition]) do |f| %>
-	<div>
-		<p>
-			<span>
-				<%= f.label :definition %>
-			</span>
-			<span>
-				<%= f.text_field :definition %>
-			</span>
-		</p>
-		<p>
-			<span>
-				<%= f.label :usage %>
-			</span>
-			<span>
-				<%= f.text_field :usage %>
-			</span>
-		</p>
-		<p>
-			<span>
-				<%= f.label :usage_translation %>
-			</span>
-			<span>
-				<%= f.text_field :usage_translation %>
-			</span>
-		</p>
-		<p>
-			<span class="submitButton">
-				<%= f.submit %>					
-			</span>	
-		</p>
-  	
-	</div>
-<% end %>
\ No newline at end of file
+		
+<label for="definition">Definition: </label><br>
+<input class="inputField" type="text" id="definition"><br>
+<label for="usage">Usage: </label><br>
+<input class="inputField" type="text" id="usage"><br>
+<label for="usageTranslation">Usage Translation: </label><br>
+<input class="inputField" type="text" id="usageTranslation"><br>
+<a class="button" onclick="submitDefinition()">Submit Definition</a>
+
+<script type="text/javascript">
+	function submitDefinition() {
+		var definitionText = $("#definition").val();
+		var usageText = $("#usage").val();
+		var usageTranslationText = $("#usageTranslation").val();
+		console.log(window.location.pathname+'/definitions');
+		if (definitionText.trim()=='') {
+			return alert('Definition cannot be empty!')
+		} else if (usageText.trim()=='') {
+			return alert('Usage cannot be empty!')
+		} else if (usageTranslationText.trim()=='') {
+			return alert('Usage Translation cannot be empty!')
+		}
+		$.post(window.location.pathname+'/definitions', {
```

Comment:
Can you just do `/definitions`? Really don't like usage of `window` like this.

There's also a gem called JSRoutes that we could consider adding